tests/instruments/qblox/test_qblox_qrm.py

```diff
@@ -394,8 +394,8 @@ def test_acquire_qprogram_results(self, qrm: QbloxQRM):
         qrm.upload_qpysequence(qpysequence=sequence, channel_id=0)
 
         qp_acqusitions = {
-            "acquisition_0": AcquisitionData(bus="readout_q0", save_adc=False),
-            "acquisition_1": AcquisitionData(bus="readout_q0", save_adc=True)
+            "acquisition_0": AcquisitionData(bus="readout_q0", save_adc=False, shape=(-1,)),
+            "acquisition_1": AcquisitionData(bus="readout_q0", save_adc=True, shape=(-1,))
```

Comment:
We should add tests that cover no shape, 1D shape and multiloop shape and check both `array()` and `threshold` methods.

Reply:
added some tests to cover that.